src/components/_channel/CollectorsBox/CollectorsBox.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
+
+import { Avatar } from '@/components/Avatar'
+import { AvatarGroup, AvatarGroupSingleAvatar } from '@/components/Avatar/AvatarGroup'
+import { ListItem } from '@/components/ListItem'
+import { Text } from '@/components/Text'
+import { SvgActionPlus } from '@/components/_icons'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { cVar } from '@/styles'
+
+import { CollectorsBoxWrapper, PlusIconBackground, PlusIconWrapper } from './CollectorsBox.styles'
+
+type Collector = AvatarGroupSingleAvatar & { nftsAmount?: number }
+
+export type CollectorsBoxProps = {
+  collectors: Collector[]
+  maxShowedCollectors?: number
+}
+
+export const CollectorsBox: React.FC<CollectorsBoxProps> = ({ collectors, maxShowedCollectors = 5 }) => {
+  const [open, setOpen] = useState(false)
+  const [isHovered, setIsHovered] = useState(false)
+
+  const sortedCollectors = [...collectors]
+    .sort((a, b) => (b?.nftsAmount || 0) - (a?.nftsAmount || 0))
+    .map((collector) => ({
+      ...collector,
+      onClick: (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
+        e.stopPropagation()
+        collector.onClick?.(e)
+      },
+    }))
+
+  const avatarsWhenCollectorsMoreThan5: Collector[] = [
```

Comment:
Lets come up with a better name since this will not always be 5, maybe avatarsWhenCollectorsAreTooMany